language/api/README.md

```diff
@@ -0,0 +1,87 @@
+
+# Google Cloud Natural Language API Sample
+
+This Python sample demonstrates the use of the [Google Cloud Natural Language API][NL-Docs]
+for sentiment, entity, and syntax analysis.
+
+[NL-Docs]: https://cloud.google.com/natural-language/docs/
+
+## Setup
+
+Please follow the [Set Up Your Project](https://cloud.google.com/natural-language/docs/getting-started#set_up_your_project)
+steps in the Quickstart doc to create a project and enable the
+Cloud Natural Language API. Following those steps, make sure that you
+[Set Up a Service Account](https://cloud.google.com/natural-language/docs/common/auth#set_up_a_service_account),
+and export the following environment variable:
+
+```
```

Comment:
Not sure. See my comment [here](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/275#discussion_r71449047), though..